src/mantarray_desktop_app/main.py

```diff
@@ -127,6 +127,13 @@ def _log_system_info() -> None:
         logger.info(msg)
 
 
+class Filter_System_Status_Logs(logging.Filter):
```

Comment:
We're going to remove this class most likely since this isn't the correct way to handle logging related to flask routes, but keep in mind this is the incorrect casing for python classes. Please check the definitions of other classes in this or other python repos for the correct casing.